### ISSUE TITLE
`no-multi-asterisks` false positive on single line doc

### ISSUE THREAD

**Issue by OpportunityLiu** [gajus/eslint-plugin-jsdoc#735]
## Expected behavior

no error for single line doc
## Actual behavior

`no-multi-asterisks` for single line doc
## ESLint Config

```yml
    parser: '@typescript-eslint/parser'
    parserOptions:
      project: tsconfig.json
    extends:
      - eslint:recommended
      - plugin:@typescript-eslint/recommended
      - plugin:@typescript-eslint/recommended-requiring-type-checking
      - plugin:jsdoc/recommended
    rules:
      jsdoc/require-jsdoc:
        - 1
        - require:
            ClassDeclaration: true
            FunctionDeclaration: true
          contexts:
            - TSMethodSignature
            - TSInterfaceDeclaration
            - TSTypeAliasDeclaration
            - MethodDefinition[ kind=/(method|get)/ ]:has(FunctionExpression)
            - { context: ClassProperty, inlineCommentBlock: true }
            - { context: TSInterfaceBody > TSPropertySignature, inlineCommentBlock: true }
      jsdoc/no-types: 1
      jsdoc/require-description:
        - 1
        - contexts: [any]
      jsdoc/require-param: 0
      jsdoc/require-param-type: 0
      jsdoc/require-returns: 0
      jsdoc/require-returns-check: 0
      jsdoc/require-returns-type: 0

```

## ESLint sample

```js
/** foo */
function foo(): void {
    //
}
```

## Environment

- Node version: v14.17.0
- ESLint version v7.26.0
- `eslint-plugin-jsdoc` version: 34.6.0

**Comment by brettz9** [comment]
Thanks for the report, and apologies for overlooking that important case!
